src/pages/Cart/cart.jsx

```diff
@@ -1,11 +1,76 @@
-import React from 'react';
+import React, { useState } from 'react';
+
+import './cart.css'
+import { Header, Footer, CartCard, EmptySection } from '../../components'
+import { emptyBag } from '../../assets/images';
+import { useCart } from '../../context'
 
 function CartPage(){
+    
+    const { cartItems, setCartItems } = useCart()
+    let cartProductsCount = cartItems.length
+    let totalAmount = cartItems ? cartItems.reduce((acc, curr) => acc + curr.price * curr.cartCount, 0) : 0;

```

Comment:
a minor suggestion - you may abstract these methods in a util file.

Reply:
sure, will do !